util/dvsim/FlowCfg.py

```diff
@@ -83,10 +84,16 @@ def __init__(self, flow_cfg_file, proj_root, args):
         self.results_summary_server_html = ""
         self.results_summary_server_page = ""
 
-        # Full and summary results in md text.
+        # Full results in md text
         self.results_md = ""
+        # Selectively sanitized md results to be mailed out or published
+        self.email_results_md = ""
+        self.publish_results_md = ""
+        self.sanitize_email_results = ""
```

Comment:
I think these are better initialized to `False` (once you make the HJson change above).

Reply:
jep, done.